File: java/test/_polytrack3.java
import java.util.Arrays;
import liba.polytrack3;

class _polytrack3 {
    public static void main(String[] args) {
        polytrack3 obj = new polytrack3(0, 1, 0, 1, 0, 1);
        double[] out = obj.out(0.5);
        System.out.println(Arrays.toString(obj.k()));
        System.out.println(obj.pos(0.5) + "," + obj.vel(0.5) + "," + obj.acc(0.5));
        System.out.println(out[0] + "," + out[1] + "," + out[2]);
    }
}

File: quickjs/test/polytrack3.js
import * as a from "liba.so";
var polytrack3 = new a.polytrack3(0, 1, 0, 1)
polytrack3.gen(0, 10, 0, 10, 0, 0)
for (var i = 0; i < 10; i += 0.1) {
    console.log(i, polytrack3.pos(i), polytrack3.vel(i), polytrack3.acc(i))
}
